examples/app.js

```diff
@@ -1,36 +1,49 @@
 import QRCodeModal from "@walletconnect/qrcode-modal";
 import { WalletConnectProvider } from "../out/walletConnectProvider";
 
-export async function main() {
-    let bridge = "https://bridge.walletconnect.org";
+const BridgeUrl = "https://bridge.walletconnect.org";
```

Comment:
This should probably be pascalCase, since it's a constant

Reply:
Renamed.